scripts/monitoring/cron-send-ssl-check.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+""" check ssl cert info"""
+
+# We just want to see any exception that happens
+# don't want the script to die under any cicumstances
+# script must try to clean itself up
+# pylint: disable=broad-except
+
+# main() function has a lot of setup and error handling
+# pylint: disable=too-many-statements
+
+# main() function raises a captured exception if there is one
+# pylint: disable=raising-bad-type
+
+# Adding the ignore because it does not like the naming of the script
+# to be different than the class name
+# pylint: disable=invalid-name
+
+import argparse
+import datetime
+import random
+import string
+import time
+import urllib2
+import ssl
+import socket
+import logging
+import OpenSSL
+
+
+# Our jenkins server does not include these rpms.
+# In the future we might move this to a container where these
+# libs might exist
+#pylint: disable=import-error
+from openshift_tools.monitoring.ocutil import OCUtil
+from openshift_tools.monitoring.metric_sender import MetricSender
+
+
+logging.basicConfig(
+    format='%(asctime)s - %(relativeCreated)6d - %(levelname)-8s - %(message)s',
+)
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+ocutil = OCUtil()
+
+def parse_args():
+    """ parse the args from the cli """
+    logger.debug("parse_args()")
+
+    parser = argparse.ArgumentParser(description='OpenShift public ssl expired info check ')
+    parser.add_argument('-v', '--verbose', action='store_true', default=None, help='Verbose?')
+    parser.add_argument('-l', '--list', nargs='+', help='domain that need to check', required=True)
+    return parser.parse_args()
+
+def send_metrics(day_left):
+    """ send data to MetricSender"""
+    logger.debug("send_metrics()")
+
+    ms_time = time.time()
+    ms = MetricSender()
+    logger.info("Send data to MetricSender")
+    ms.add_metric({'openshift.master.public.ssl.left': day_left})
```

Comment:
`ms.add_metric({'openshift.master.public.ssl.left': day_left})`

Can this key `openshift.master.public.ssl.left` be passed in via arguments, so we can reuse the script for different SSL certs and change the key dynamically